sentry-ruby/spec/sentry/transport_spec.rb

```diff
@@ -531,11 +531,14 @@
         end
       end
 
-      context "with Faraday::Error" do
+      context "with an HTTP exception" do
+        before do
+          stub_request(:post, "http://sentry.localdomain:80/sentry/api/42/envelope/").
+          to_raise(Timeout::Error)
+        end
```

Comment:
I do not know what the intention of this spec was originally since Faraday was NOT configured here and the spec was passing by an accident since there was a socket timeout raised under the hood.

I can add a faraday spec too. Just lemme know if it'd make sense.

Reply:
its a leftover from the legacy Raven SDK, ignore
https://github.com/getsentry/sentry-ruby/blob/cc1dc53c0d60a21db7ad95cac05590ec23c82910/sentry-raven/lib/raven/transports/http.rb#L28

Reply:
Yes we may just remove the entire test case.

Reply:
I'm gonna leave it. It does verify specific error handling - HTTPTransport defines a list of exceptions that are rescued in send_data and now this spec checks that it works as expected that the exception is wrapped in Sentry::ExternalError.